.github/workflows/python-package.yml

```diff
@@ -57,6 +56,12 @@ jobs:
     - uses: actions/checkout@v2
       with:
         submodules: recursive
+        fetch-depth: '0'
+    - name: Check for [skip ci] in commit message
+      uses: mstachniuk/ci-skip@v1
+      with:
+        fail-fast: true
+        commit-filter: 'skip ci'
```

Comment:
I think the square brackets should be part of the filter to avoid false positives.